js/fileupload.js

```diff
@@ -130,8 +130,8 @@ var handleUploadedFile = function (files, files_data, input_name, container, edi
  * @param      {String}  input_name    Name of generated input hidden (default filename)
  * @param      {Object}  container     The fileinfo container
  */
-var fileindex = 0;
 var displayUploadedFile = function(file, tag, editor, input_name, filecontainer) {
+    var fileindex = $('input[name^="_'+input_name+'["]').length;
```

Comment:
I may be wrong, but I think the `[` that is inside the value has to be escaped.
```suggestion
    var fileindex = $('input[name^="_'+input_name+'\["]').length;
```

Reply:
I did the following test 

I have a file upload field with 2 filles in the DOM.
I duplicated an input and edited it to have it not indexed

![image](https://user-images.githubusercontent.com/14139801/228786746-ffb27dd1-d7d5-4f27-b988-7997ba83cf8a.png)

Then I tried to count the inputs
![image](https://user-images.githubusercontent.com/14139801/228787047-38996d24-ec4c-4cae-aca8-8740729e51fa.png)

Without the opening bracket, i find 3 inputs, as expected. If I add the bracked, with and without escaping, the count is 2, as expected. I think this is not necessary to escape it.

Reply:
Indeed, I guess it is not required as value is inside quotes.